<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSnapshotDetails</class>
 <widget class="QMainWindow" name="DlgSnapshotDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snapshot details</string>
  </property>
	<widget class="QWidget" name="centralWidget">
	 <layout class="QVBoxLayout" name="verticalLayout">
		<property name="leftMargin">
		 <number>0</number>
		</property>
		<property name="topMargin">
		 <number>0</number>
		</property>
		<property name="rightMargin">
		 <number>0</number>
		</property>
		<property name="bottomMargin">
		 <number>0</number>
		</property>
		<item>
		 <layout class="QGridLayout" name="gridLayout">
			<property name="sizeConstraint">
			 <enum>QLayout::SetDefaultConstraint</enum>
			</property>
			<item row="1" column="0">
			 <widget class="QLabel" name="lHeapSize">
				<property name="sizePolicy">
				 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
					<horstretch>0</horstretch>
					<verstretch>0</verstretch>
				 </sizepolicy>
				</property>
				<property name="text">
				 <string>Heap size (KiB)</string>
				</property>
				<property name="alignment">
				 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				</property>
			 </widget>
			</item>
			<item row="0" column="1">
			 <widget class="QLabel" name="txtTimestamp">
				<property name="sizePolicy">
				 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
					<horstretch>0</horstretch>
					<verstretch>0</verstretch>
				 </sizepolicy>
				</property>
				<property name="text">
				 <string>0</string>
				</property>
				<property name="alignment">
				 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				</property>
			 </widget>
			</item>
			<item row="1" column="1">
			 <widget class="QLabel" name="txtHeapSize">
				<property name="sizePolicy">
				 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
					<horstretch>0</horstretch>
					<verstretch>0</verstretch>
				 </sizepolicy>
				</property>
				<property name="text">
				 <string>0</string>
				</property>
				<property name="alignment">
				 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				</property>
			 </widget>
			</item>
			<item row="2" column="1">
			 <widget class="QLabel" name="txtHeapExtraSize">
				<property name="sizePolicy">
				 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
					<horstretch>0</horstretch>
					<verstretch>0</verstretch>
				 </sizepolicy>
				</property>
				<property name="text">
				 <string>0</string>
				</property>
				<property name="alignment">
				 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				</property>
			 </widget>
			</item>
			<item row="0" column="0">
			 <widget class="QLabel" name="lTimestamp">
				<property name="sizePolicy">
				 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
					<horstretch>0</horstretch>
					<verstretch>0</verstretch>
				 </sizepolicy>
				</property>
				<property name="text">
				 <string>Timestamp</string>
				</property>
				<property name="alignment">
				 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				</property>
			 </widget>
			</item>
			<item row="2" column="0">
			 <widget class="QLabel" name="lHeapExtraSize">
				<property name="sizePolicy">
				 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
					<horstretch>0</horstretch>
					<verstretch>0</verstretch>
				 </sizepolicy>
				</property>
				<property name="text">
				 <string>Heap extra size (KiB)</string>
				</property>
				<property name="alignment">
				 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				</property>
			 </widget>
			</item>
			<item row="0" column="2">
			 <spacer name="horizontalSpacer">
				<property name="orientation">
				 <enum>Qt::Horizontal</enum>
				</property>
				<property name="sizeHint" stdset="0">
				 <size>
					<width>40</width>
					<height>20</height>
				 </size>
				</property>
			 </spacer>
			</item>
			<item row="1" column="2">
			 <spacer name="horizontalSpacer_2">
				<property name="orientation">
				 <enum>Qt::Horizontal</enum>
				</property>
				<property name="sizeHint" stdset="0">
				 <size>
					<width>40</width>
					<height>20</height>
				 </size>
				</property>
			 </spacer>
			</item>
			<item row="2" column="2">
			 <spacer name="horizontalSpacer_3">
				<property name="orientation">
				 <enum>Qt::Horizontal</enum>
				</property>
				<property name="sizeHint" stdset="0">
				 <size>
					<width>40</width>
					<height>20</height>
				 </size>
				</property>
			 </spacer>
			</item>
		 </layout>
		</item>
		<item>
		 <widget class="AllocationsGraph" name="grAllocations" native="true">
			<property name="sizePolicy">
			 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			 </sizepolicy>
			</property>
		 </widget>
		</item>
	 </layout>
	</widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AllocationsGraph</class>
   <extends>QWidget</extends>
   <header>allocationsgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
